tests/integration/security/sds_ingress/single_mtls_gateway/single_mtls_gateway_test.go

```diff
@@ -105,5 +105,41 @@ func TestSingleMTLSGateway_SecretRotation(t *testing.T) {
 			if err != nil {
 				t.Errorf("unable to retrieve 200 from product page at host %s: %v", host, err)
 			}
+
```

Comment:
I personally prefer the each test is focused on one feature. TestBasicSDS, TestCertificateRevocation...
So that we have finer understanding in case of a test fail.

Reply:
Optional, I'll leave it up to you and @JimmyCYJ though, since this test is already written in a sequential way...

might worth add a TODO

Reply:
This test is too long, could you move the part into a separate test? TestCertificateRevocation sounds good.